tket/src/Ops/Op.cpp

```diff
@@ -35,7 +35,9 @@ std::string Op::get_command_str(const unit_vector_t& args) const {
   if (!args.empty()) {
     out << " " << args[0].repr();
     for (unsigned i = 1; i < args.size(); i++) {
-      out << ", " << args[i].repr();
+      if (args[i].type() != UnitType::WASMUIDT) {
```

Comment:
How much do we want to hide the wasm wire from the public? Happy to discuss this.

Reply:
I think it should be visible. In future we will probably want to allow multiple WASM wires in a circuit, and we'd want it to be visible which commands were attached to which ones.

Reply:
My main argument for hiding it for now is that a lot of this functions are interacting with the qasm generation where we need to hide them. Or are there any plans to have them shown in the extended qasm somehow?